hadoop-hdfs-project/hadoop-hdfs/dev-support/findbugsExcludeFile.xml

```diff
@@ -310,4 +310,10 @@
       <Method name="reconcile" />
       <Bug pattern="SWL_SLEEP_WITH_LOCK_HELD" />
     </Match>
+    <!-- Manually verified that r can never be null and cannot result in NPE -->
```

Comment:
can we revert the formatting change in findbugsExcludeFile.xml as the assert should fix the spotbugs issue?